CoffeeChat/Cells/Interest.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  Interest.swift
+//  CoffeeChat
+//
+//  Created by Phillip OReggio on 2/4/20.
+//  Copyright © 2020 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+struct Interest {
+
+    let name: String
+    let categories: String
+    let image: UIImage
+
+    init(name: String, categories: String, image: UIImage) {
+        self.name = name
+        self.categories = categories
+        self.image = image
```

Comment:
a thought: would it be more efficient to pass in a string in the init that we then turn into an image? because later when we get the interest data, we'd most likely be getting the name of the image and not the image itself so might as well standardize it here

Reply:
ok sounds good; the string would be the name of the image or a string representation of image?

Reply:
im actually not sure how we will be storing these images (esp cause we have so many interests, groups, etc. so idk if we'd just put everything is assets). for now let's just do name of the image though??